metadata/Cadastre/land_ownership

```diff
@@ -24,19 +24,21 @@ Standard colors are recommended when displaying this feature class. A layer file
 
 ### What is the purpose of the dataset?
 
-Land ownership data is used by many government and private entities to demarcate boundaries, provide evidence in legal disputes, and display administrative jurisdictions for agencies such as the Bureau of Land Management and the US Forest Service. This dataset is also a helpful resource for public recreation, news outlets, and data analysts. For example, land ownership data has been used by the [Utah Broadband Center](https://business.utah.gov/broadband/) and UGRC to determine areas with limited broadband internet access in an effort to improve services in  underserved communities. You can learn more about the Utah Broadband Project on our [blog](https://gis.utah.gov/blog/2016-02-01-utah-broadband-outreach-center-releases-revamped-availability-map/).
+Land ownership data is used by many government and private entities to demarcate boundaries, provide evidence in legal disputes, and display administrative jurisdictions for agencies such as the BLM and the US Forest Service. This dataset is also a helpful resource for public recreation, news outlets, and data analysts.
+
+For example, land ownership data has been used by the [Utah Broadband Center](https://business.utah.gov/broadband/) and UGRC to determine areas with limited internet access in an effort to improve services in underserved communities. You can learn more about the Utah Broadband Project on our [blog](https://gis.utah.gov/blog/2016-02-01-utah-broadband-outreach-center-releases-revamped-availability-map/).
```

Comment:
This sounds like what I did with address points. @jacobdadams did you use land ownership for underserved communities?

Reply:
No, but the tribal boundary data have been essential for identifying which groups should challenge the fabric or apply for funding.

Reply:
@ZachBeck In my notes from our conversation the other week, I had written down that the broadband project was for land ownership, but I must have written it down in the wrong spot since we talked about both layers. I think it's a neat application, so I'll go ahead and move it over to address points, my bad for getting it mixed up! 

@jacobdadams do you happen to have a link to a blog post or something that talks about the tribal boundaries being used for that? I think it would be awesome to include!